docs/documentation/main-concepts/repository-structure.rst

```diff
@@ -7,8 +7,10 @@
 
 .. _build-repository:
 
-Build a repository
-==================
+Repository structure
+====================
+
+CHANGE ME PLEASE
```

Comment:
Maybe a `TODO` comment with what to change, or the actual content even better

Reply:
Forgot about that, thanks for pointing it out. It was to remind us that the tone of the page did not match the rest; I updated it.